apps/wallet/src/ui/app/hooks/deepbook/useGetEstimate.ts

```diff
@@ -116,22 +117,14 @@ async function getPlaceMarketOrderTxn({
 	let txnResult;
 
 	if (isAsk) {
-		const bigNumberBaseBalance = new BigNumber(baseBalance);
+		const actualBalance = isPayAll
+			? new BigNumber(baseBalance).minus(getWalletFees(baseBalance))
```

Comment:
Updated this section to calculate via below

Basically if balance + fees > maxAvail then I will do a balance - fees. It will not be completely correct everytime, but it should handle most cases when the balance swap is large (close to or at max)